components/email/src/Email.svelte

```diff
@@ -118,17 +120,26 @@
     );
     show_star = getPropertyValue(internalProps.show_star, show_star, false);
     unread = getPropertyValue(internalProps.unread, unread, null);
+
     click_action = getPropertyValue(
       internalProps.click_action,
       click_action,
       "default",
     );
+    thread_id = getPropertyValue(internalProps.thread_id, thread_id, "");
```

Comment:
Side effect: noticed that `thread_id` from manifest was not triggering a fetch. Turns out it wasn't added to the property list here.